Documentation/source/changelog.rst

```diff
@@ -2,6 +2,17 @@
 Changelog
 =========
 
+1.0.0-alpha.1 (15-03-2018)
```

Comment:
There are some breaking changes for Swift projects, and that would imply a major version release. Not yet since I wanted this PR to be reviewed first and see if this could be done.